.gitmodules

```diff
@@ -31,12 +25,18 @@
 [submodule "src/lib/ecl"]
 	path = src/lib/ecl
 	url = https://github.com/PX4/ecl.git
-[submodule "cmake/cmake_hexagon"]
-	path = cmake/cmake_hexagon
-	url = https://github.com/ATLFlight/cmake_hexagon
 [submodule "src/drivers/gps/devices"]
 	path = src/drivers/gps/devices
 	url = https://github.com/PX4/GpsDrivers.git
 [submodule "src/lib/micro-CDR"]
 	path = src/lib/micro-CDR
 	url = https://github.com/eProsima/micro-CDR.git
+[submodule "platforms/nuttx/src/modules/uavcan/libuavcan"]
```

Comment:
The Eagle board has a DSP that runs QuRT and runs PX4 and it has an ARM processor also running PX4 and they communicate through muorb. They are one board and have one board ID.